server/index.js

```diff
@@ -30,3 +37,25 @@ app.put('/api/nearby/:carousel_id', (req, res) => {
     .then(updated => res.status(202).send(updated.carousel))
     .catch(err => res.status(400).json(err));
 });
+
+app.put('/api/nearby/:carousel_id', (req, res) => {
+  db.addFavorite(req.params.carousel_id, req.query.restaurantId, req.query.increment)
+    .then(updated => res.status(202).send(updated.carousel))
+    .catch(err => res.status(400).json(err));
+});
+
+app.put('/api/nearbyUpdate/', (req, res) => {
+  const carouselUpdate = req.body;
+  const { id, carousel } = carouselUpdate;
+  db.updateCarouselById(id, carousel)
+    .then((result) => {
+      res.status(200).send(result);
+    })
+    .catch(err => res.status(400).json(err));
+});
+
+app.delete('/api/nearby/:carousel_id', (req, res) => {
+  db.deleteCarouselById(req.params.carousel_id)
+    .then(result => res.status(200).send(result))
```

Comment:
On some REST API standards, you respond to a DELETE request with the deleted item. You might want to consider doing that here (though it's not like there's an official REST API standard or anything like that).

Reply:
Great suggestion, done.